front-end/src/components/Admin/Grid/index.js

```diff
@@ -100,16 +102,33 @@ class Grid extends Component {
     const { entityType, tabKey } = this.props.match.params;
     const { setTabContentUrl } = this.props;
     const { currentPage, pagesTotal } = this.state.meta;
+    const paginationPages = [];
+
     setTabContentUrl(entityType);
 
+    for (let number = 1; number <= pagesTotal; number++) {
```

Comment:
We must not show a Pagination.Item for each page. Image that we have 1000 pages. They won't fit the screen. We need to show 5-7 nearest pages. E.g. if we're on page 8, we should see < 6 7 8 9 10 >